### ISSUE TITLE
Build output map with empty war3map.j, on Raspberry Pi

### ISSUE THREAD

**Issue by jmrico01** [wurstscript/WurstScript#978]
### Describe the bug
I recently got a Raspberry Pi 4, and decided it would be fun to use it as a CI runner for Wurst map builds. Since this is Java, it must simply work anywhere, right? Turns out, it almost works! However, it seems the output map has an empty `war3map.j`, even though the code is correctly compiled and saved in `temp/output.j`.

Environment is a Raspberry Pi 4 (ARM linux), with Java version:
```
openjdk 11.0.9.1 2020-11-04
OpenJDK Runtime Environment (build 11.0.9.1+1-post-Raspbian-1deb10u2)
OpenJDK Server VM (build 11.0.9.1+1-post-Raspbian-1deb10u2, mixed mode)
```

Starting only with my map project and `WurstSetup.jar`:
1. `java -jar WurstSetup.jar install wurstscript`
1. `java -jar ~/.wurst/WurstSetup.jar install`
1. `java -jar ~/.wurst/WurstSetup.jar build base.w3x`
1. I move the output map in `_build` to a Windows machine and inspect with an MPQ editor. Everything looks good except `war3map.j` is 0 bytes. `temp/output.j` has all the code, though...

[compiler.log](https://github.com/wurstscript/WurstScript/files/5657106/compiler.log)

### Speculation
When I do a Windows build, `compiler.log` at first is similar/equal to the one in the Pi. Afterward, it gets replaced with one where, in the final MPQ editing section, only has `Added file war3map.j ` and a few others. This suggests to me that there are 2 MPQ editing/packing passes in the Wurst build, and it's possible that the 2nd one just isn't running on the Pi. There is no `Added file war3map.j ` log there.

**Comment by Frotty** [comment]
You should use the package manager commands `grill build` as described in the CI docker image https://hub.docker.com/r/frotty/wurstscript

**Comment by jmrico01** [comment]
Just tried `~/.wurst/grill build base.w3x`, but it didn't seem to change anything.

**Comment by Frotty** [comment]
I misread the original post and thought u were calling wurstscript.jar instead of setup/grill. So except for some suspicious runargs it seems fine and should work. Can you check if `grill build` works correctly on your normal (windows?) machine to verify that it's a problem on the rasp only?

**Comment by jmrico01** [comment]
Yep, been doing `grill build base.w3x` on my Windows machine, and everything works as expected. In the Pi, the `war3map.j` is 0 bytes but everything else is OK.

**Comment by jmrico01** [comment]
Yeah that sounds right. Weird because everything else gets inserted - I have an `imports` folder with a bunch of files, and those end up in the map no problem. My current workaround is to have a post-build step that inserts the `temp/output.j` into the built map as `war3map.j`.

**Comment by Frotty** [comment]
any update on this?

**Comment by jmrico01** [comment]
Not exactly the same environment, but I recently tried this on a new mac with an Apple silicon chip (ARM 64bit) and saw no issues. I also had to fix some pjass errors before the build fully worked, so now I'm wondering if it was those errors causing the issue to begin with. I can re-test on my raspberry pi in a few weeks.

**Comment by Frotty** [comment]
Okay, might also have been fixed in the meantime. Some test case would be helpful.
